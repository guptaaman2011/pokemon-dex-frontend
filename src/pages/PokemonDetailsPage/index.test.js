import React from "react";
import { shallow } from "enzyme";
import { shallowToJson } from "enzyme-to-json";
import { PokemonDetailsPage } from "..";

describe("PokemonDetailsPage", () => {
  it("should render without breaking", () => {
    const wrapper = shallow(<PokemonDetailsPage />);
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});
