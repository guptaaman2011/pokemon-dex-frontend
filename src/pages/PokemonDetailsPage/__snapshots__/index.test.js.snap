// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PokemonDetailsPage should render without breaking 1`] = `
<div>
  <h2>
    PokemonDetailsPage
  </h2>
</div>
`;
